pyfr/solvers/baseadvec/inters.py

```diff
@@ -94,3 +98,33 @@ def _eval_opts(self, opts, default=None):
             return [npeval(cfg.get(sect, k, default), cc) for k in opts]
         else:
             return [npeval(cfg.get(sect, k), cc) for k in opts]
+
+    def _exp_opts(self, opts, lhs, default=None):
```

Comment:
I tested the default option settin the default u, v, w to zero when not
specified in sub-in-frv. I also modified the exp_option to be able to take
different defaults for different options. I am currently testing these,
will report back.

On Thu, May 14, 2015 at 10:48 PM, Freddie Witherden <
notifications@github.com> wrote:

> In pyfr/solvers/baseadvec/inters.py
> <https://github.com/vincentlab/PyFR/pull/52#discussion_r30366865>:
>
> > @@ -94,3 +98,33 @@ def _eval_opts(self, opts, default=None):
> >              return [npeval(cfg.get(sect, k, default), cc) for k in opts]
> >          else:
> >              return [npeval(cfg.get(sect, k), cc) for k in opts]
> > +
> > +    def _exp_opts(self, opts, lhs, default=None):
>
> Okay. If you can do some local testing to ensure that everything works as
> expected when default is not None then I'm fine with it. (I am always
> cautious about comitting code that might be untested.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vincentlab/PyFR/pull/52/files#r30366865>.
>



-- 
================================
Dr. Iyer Arvind Sundaram

The reasonable man adapts himself to the world; the unreasonable one
persists in trying to adapt the world to himself.

Therefore all progress depends on the unreasonable man

- George Bernard Shaw.